app/views.py

```diff
@@ -1,22 +1,54 @@
-from flask import render_template, request
+from flask import render_template, request, redirect, url_for
 
 from app import app
 
+users = set()
+users_bucketlist = []
+
 @app.route('/')
 def index():
     return render_template('home.html')
 
-@app.route('/login')
-def login():
-    return render_template('login.html')
 
-@app.route('/signup')
+
+@app.route('/signup', methods=['GET', 'POST'])
 def signup():
+    if request.method == 'POST':
+        username = str(request.form.get('inputName'))
+        email = str(request.form.get('inputEmail'))
+        password = str(request.form.get('inputPassword'))
+
+        if username:
+            users.add(username)
+            users.add(email)
+            users.add(password)
+            print users
+            return redirect(url_for('login'))
     return render_template('signup.html')
 
+@app.route('/login', methods=['GET', 'POST'])
+def login():
+    if request.method == 'POST':
+        email = str(request.form.get('inputEmail'))
+        password = str(request.form.get('inputPassword'))
+        if email in users and password in users:
```

Comment:
Storing data in the set still implies two people cannot have the same password. Kindly review the appropriate use of data stuctures. I would recommend you check the use of objects.